priv/docker-compose.geth-ganache.yml

```diff
@@ -11,7 +11,7 @@ services:
       - ETHEREUM_WS_RPC_URL=ws://geth:8545
   plasma-contracts:
     environment:
-      - MIN_EXIT_PERIOD=1
+      - MIN_EXIT_PERIOD=40
```

Comment:
40? why?

Reply:
We need this extended so that piggybacking can occur within the "first phase" of the exit period.

Reply:
20 wouldn't work? I'm more interested in how did you decide on the number...

Reply:
It's blockchain, so he started at 1 and went his way up until it worked... 🤷‍♂ On a more serious note, if you IFE a tx and want to piggyback each output (or input if non-canonical), it's going to take quite a bit of time :/

Reply:
At first I LOLed and now I'm crying.